311-status-protocol.md

```diff
@@ -0,0 +1,262 @@
+---
+id: 311-status-protocol
+title: Protocol Engineering Swarm
+---
+
+# Protocol Engineering Swarm
+
+Draft, 2018-12-01.
+
+## Summary
+This swarm will research and develop a set of open protocols that is a
+reflection of our [our principles](https://our.status.im/our-principles/).
+
+This set of protocols will as a whole enable the implementation of the Status
+whitepaper, and similar use cases, through various clients. Part of the effort
+will be about selecting existing protocols and standards, when appropriate, and
+part of it will be about developing new protocols to meet our needs.
+
+Initial focus is on providing an anonymous communication protocol, which is
+developed in conjunction with other teams in the Web3 / secure messaging space.
+
+## Goals
+
+1. Create a set of open protocol(s) for secure messaging that reflects our
```

Comment:
a small inconsistency: "secure messaging" here, "communication protocol" in other places